lms/static/js/groups/models/cohort_discussions.js

```diff
@@ -0,0 +1,10 @@
+var edx = edx || {};
+
+(function(Backbone) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.DiscussionTopicsModel = Backbone.Model.extend({
+    });
```

Comment:
Can you add the parameters that will be set on this model? Helps in understanding.